edm4eic.yaml

```
@@ -370,6 +370,15 @@ datatypes:
     OneToManyRelations:
       - edm4eic::TrackerHit hits            // Hits in this measurement (single or clustered)
 
+  edm4eic::TrackSeed:
+    Description: "save seed info from the realistic seed finder"
+    Author: "S. Li, B. Schmookler, J. Osborn"
+    Members:
+      - edm4hep::Vector3f  perigee   // vector to create the perigee (line surface)
+    OneToManyRelations:
+      - edm4eic::TrackerHit triplet  // three tracker hits that are used to form the seed 
```

```suggestion
      - edm4eic::TrackerHit hits  // tracker hits that are used to form the seed 
```
Three is an arbitrary number.